cmd/clusterctl/client/cluster/objectgraph_test.go

```diff
@@ -1127,15 +1107,15 @@ func Test_objectGraph_setSoftOwnership(t *testing.T) {
 				sort.Strings(wantObjects)
 				sort.Strings(gotObjects)
 
-				if !reflect.DeepEqual(gotObjects, wantObjects) {
-					t.Fatalf("secret %s, got = %s, expected = %s", secret.identity.UID, gotObjects, wantObjects)
-				}
+				g.Expect(gotObjects).To(Equal(wantObjects))
```

Comment:
```suggestion
				g.Expect(gotObjects).To(ConsistOf(wantObjects))
```
This also removes the need to call `sort.Strings()` above for `wantObjects` and `gotObjects`